src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -1180,6 +1180,19 @@
         --set safetyProfile.allowDeletionOfReplicatedLocations=true
 """
 
+helps['sig image-version undelete'] = """
+type: command
+short-summary: Restore soft deleted Image Version
+long-summary: >
+    Operation will only be successful if used within a Gallery with soft-deletion on.
+examples:
+  - name: Restore a soft deleted Image Version
+    text: |
+        az sig image-version undelete --resource-group MyResourceGroup \\
+        --gallery-name MyGallery --gallery-image-definition MyImage \\
+        --gallery-image-version 1.1.1
```

Comment:
@kangsun-ctrl Could you please see if the design of this command meets your expectations?